af/fastpath/fastpath/db.py

```diff
@@ -156,3 +168,158 @@ def upsert_summary(
         # notification_json = ujson.dumps(notification)
         # q = f"SELECT pg_notify('fastpath', '{notification_json}');"
         # cur.execute(q)
+
+
+# # Clickhouse
+
+
+def query_click(query, query_params):
+    q = click_client.execute(query, query_params, with_column_types=True)
+    rows, coldata = q
+    colnames, coltypes = tuple(zip(*coldata))
+
+    for row in rows:
+        yield dict(zip(colnames, row))
+
+
+def _click_create_table_fastpath():
+    # TODO: table creation should be done before starting workers
+    sql = """
+    CREATE TABLE IF NOT EXISTS fastpath
+    (
+        `measurement_uid` String,
+        `report_id` String,
+        `input` String,
+        `probe_cc` String,
+        `probe_asn` Int32,
+        `test_name` String,
+        `test_start_time` DateTime,
+        `measurement_start_time` DateTime,
+        `filename` String,
+        `scores` String,
+        `platform` String,
+        `anomaly` String,
+        `confirmed` String,
+        `msm_failure` String,
+        `domain` String,
+        `software_name` String,
+        `software_version` String,
+        `control_failure` String,
+        `blocking_general` Float32,
+        `is_ssl_expected` Int8,
+        `page_len` Int32,
+        `page_len_ratio` Float32,
+        `server_cc` String,
+        `server_asn` Int8,
+        `server_as_name` String
```

Comment:
Maybe since we are adding to the base schema fields for the future, we can also add here `test_runtime`?